### PR TITLE
Updated campaigns documentation.

### PR DESCRIPTION
**Pull request by angaither**
#### What's this PR do?

pulls all campaign endpoints into `campaigns.md` & updates the README in the documentation folder
#### How should this be reviewed?

👀 
#### Any background context you want to provide?

seems like all the `""` went away. @DFurnes how u feel
#### Relevant tickets

Refs #6490
#### Checklist
- [X] Documentation added for new features/changed endpoints.

### CODE REVIEW

**Comment by DFurnes** [comment]
screen shot 2016-06-09 at 5 43 04 pm

😏

File: documentation/endpoints/campaigns.md [hunk 1]
```diff
@@ -1,250 +1,305 @@
-# Campaigns Endpoint
+# Campaigns
 
-- [Retrieve A Campaign](#retrieve-a-campaign)
-
-## Retrieve A Campaign
-
-Get data for a specified campaign.
+## Retrieve all campag
```
Comment (DFurnes) [inline documentation/endpoints/campaigns.md]: Okay now I'm just being an asshole but: `campag`

File: documentation/endpoints/campaigns.md [hunk 2]
```diff
@@ -289,4 +344,4 @@ curl https://www.dosomething.org/api/v1/campaigns/23/reportback -X POST
 Example response:
 ````
 ["127"]
-````
\ No newline at end of file
+````x
```
Comment (DFurnes) [inline documentation/endpoints/campaigns.md]: ❌

Comment (angaither) [inline documentation/endpoints/campaigns.md]: 👀

Comment (angaither) [inline documentation/endpoints/campaigns.md]: it's to mark the spot

**Comment by DFurnes** [comment]
🚢 🚢 📝

**Pull request merged**